paimon-filesystems/paimon-s3/pom.xml

```diff
@@ -39,6 +39,12 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>com.amazonaws</groupId>
```

Comment:
We should not rely on asw in paimon-s3.
All s3 dependencies should just be in paimon-s3-impl.

Reply:
Then how do we check argurments via `AWSCredentials`?